servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherFlatMapMerge.java

```diff
@@ -588,7 +599,7 @@ public void onNext(@Nullable final R r) {
 
             @Override
             public void onError(final Throwable t) {
-                if (!parent.source.delayError) {
+                if (parent.source.maxDelayedErrors <= 0) {
```

Comment:
Values < 0 are not allowed, consider using `parent.source.maxDelayedErrors == 0` to make it more clear what `0` value means.